core-aam/aria-readonly_true_on-spinbutton-manual.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-readonly=true on spinbutton</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "element" : "test",
+         "test" : {
+            "UIA" : [
+               [
+                  "property",
+                  "RangeValue.IsReadOnly",
+                  "is",
+                  "true"
+               ]
```

Comment:
Per the discussion on the PR against core-aam, this should also have AriaProperties.readonly exposed

```suggestion
               ],
               [
                  "property",
                  "AriaProperties.readonly",
                  "is",
                  "true"
               ]
```